### ISSUE TITLE
1024 model taking a lot of time

### ISSUE THREAD

**Issue by TrickyJustice** [arthur-qiu/LongerCrafter#12]
Hello. I am trying to inference from 1024 model and it's taking a lot of time. My code has been running for more than an hour now on a 40gb A100 GPU and this is what it displays:
Global seed set to 123
AE working on z of shape (1, 4, 64, 64) = 16384 dimensions.
>>> model checkpoint loaded.
[rank:0] 2/2 samples loaded.
[rank:0] batch-1 (1)x3 ...
DDIM scale True
ddim device cuda:0
/home/a2soni/.conda/envs/freenoise/lib/python3.8/site-packages/torch/utils/checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
DDIM scale True
ddim device cuda:0

I tried the vanilla videocrafter 1024 and it generates samples in 122 seconds. Can you help me with this?

**Comment by arthur-qiu** [comment]
1024 is really time consuming. You can reduce the sample number from 3 to 1.
